components/brave_wallet_ui/utils/buy-asset-url.ts

```diff
@@ -0,0 +1,17 @@
+import { NetworkOptionsType, AssetOptionType, UserAccountType } from '../constants/types'
+export function BuyAssetUrl (network: NetworkOptionsType, asset: AssetOptionType, account: UserAccountType, buyAmount: string) {
+  switch (network.id) {
+    case 0:
+      return `https://pay.sendwyre.com/?dest=ethereum:${account.address}&destCurrency=${asset.symbol}&amount=${buyAmount}&accountId=AC_MGNVBGHPA9T&paymentMethod=debit-card`
```

Comment:
Let's move magic values like `AC_MGNVBGHPA9T` to a separate variable, or someplace where it can be easily found.

Reply:
Moved it to its own variable! 😊